tests/components/recorder/db_schema_23_with_newer_columns.py

```diff
@@ -254,7 +254,7 @@ class States(Base):  # type: ignore
         TIMESTAMP_TYPE, default=time.time
     )  # *** Not originally in v23, only added for recorder to startup ok
     last_updated = Column(DATETIME_TYPE, default=dt_util.utcnow, index=True)
-    last_updated_ts = Column(
+    last_updated_ts = Column(  # noqa: PIE794
```

Comment:
This is already defined in line 253, but without `index=True`. I added the `noqa` because I don't know about the side-effects.